packages/lab/src/radio-button/RadioButton.tsx

```diff
@@ -70,31 +70,26 @@ export const RadioButton = forwardRef<HTMLLabelElement, RadioButtonProps>(
     });
 
     const handleFocus: FocusEventHandler<HTMLElement> = (event) => {
-      if (onFocus) {
-        onFocus(event);
-      }
+      onFocus && onFocus(event);
     };
 
     const handleBlur: FocusEventHandler<HTMLElement> = (event) => {
-      if (onBlur) {
-        onBlur(event);
-      }
+      onBlur && onBlur(event);
     };
 
     const handleInputChange: ChangeEventHandler<HTMLInputElement> = (event) => {
       const newChecked = event.target.checked;
       setCheckedState(newChecked);
-      if (onChange) {
-        onChange(event);
-      }
-      if (radioGroup && radioGroup.onChange) {
-        radioGroup.onChange(event);
+
+      onChange && onChange(event);
+
+      if (Object.keys(radioGroup).length) {
+        radioGroup.onChange && radioGroup.onChange(event);
       }
     };
 
     return (
-      <Label
-        disabled={disabled}
+      <label
```

Comment:
how come the change?

Reply:
Label has a font-size of 11px. Easier to use the label element and apply the disabled colour

Reply:
what does it need to be?` --salt-text-label-fontSize` varies by density right, so i assume those should fit?

Reply:
ah i have just seen its 11 12 14 16. i will message ben about this